doc/sphinx/proof-engine/ltac.rst

```diff
@@ -2317,11 +2317,10 @@ performance issue.
 
 .. coqtop:: reset in
 
-   Set Warnings "-omega-is-deprecated".
```

Comment:
Actually, `omega` already emits a deprecation warning, as hinted by this line (added in #11976).

Reply:
Ah, I missed that.  I was using CoqIDE 8.12+beta1, which showed the deprecation after 30 lines of stack trace for "Omega can't solve this system".  I can back out the deprecation on the tactic itself.  You think it's worth deprecating the omega flags and the `Require Import Omega`?  And still doing the removal in 8.15 instead of 8.14?

I see ci-compert has `-w -omega-is-deprecated` in their build script.  Perhaps they would turn that on and make the needed changes?